eng/config/globalconfigs/NonShipping.globalconfig

```diff
@@ -18,9 +18,6 @@ dotnet_diagnostic.CA2213.severity = suggestion
 # CA2227: Change 'property' to be read-only by removing the property setter
 dotnet_diagnostic.CA2227.severity = suggestion
 
-# RS0030: The symbol 'symbol' is banned in this project
```

Comment:
Hmm, scenario does this manifest in?

Reply:
Some test projects overrode the build to exclude analyzers, so I missed several cases in my previous pull request. The customization was removed in #3087, and now I'm cleaning up the remaining cases.